core/client/src/main/java/alluxio/client/file/options/InStreamOptions.java

```diff
@@ -30,6 +32,7 @@
 @PublicApi
 @NotThreadSafe
 public final class InStreamOptions {
+  // TODO(peis): Rename this to mCacheLocationPolicy in 2.0.
```

Comment:
you can still introduce `mCacheLocationPolicy`, but redirect old getter/setter of `mLocationPolicy` to work on `mCacheLocationPolicy`, and mark them `deprecated`